streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountProcessorDemo.java

```diff
@@ -99,7 +103,65 @@ public void close() {}
         }
     }
 
-    public static void main(String[] args) {
+    static class MyProcessorSupplierNew implements ProcessorSupplier<String, String> {
```

Comment:
the example will be reverted -- just use this for testing the upgrade -- will write unit/integration test for this later on